masterlist.yaml

```
@@ -7751,4 +7751,22 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/fallout4/mods/68889/'
         name: 'Extra Icons for FIS'
-    after: [ 'FIS-Naming-Weap-Armo-EN.esp' ]
+    after: [ 'FIS-Naming-Weap-Armo-EN.esp' ]
+
+  - name: 'sakhalinwasteland.esp'
+url:
+  - link: 'https://www.nexusmods.com/fallout4/mods/34945'
+    name: 'Fallout Sakhalin'
+  - link: 'https://mods.bethesda.net/en/fallout4/mod-detail/4081112'
+    name: 'Fallout Sakhalin'
+msg:
+  - type: say
+    content: 'Removes terrain in Far Harbor DLC'
```

* More indentation errors
* Better to add a `lang` node to make the message easier to find for translators
* I've tweaked the message

```suggestion
    msg:
      - type: say
        content:
          - lang: en
            text: 'This plugin removes terrain in the Far Harbor DLC.'
```